drivers/wifi/siwx91x/siwx91x_wifi.c

```diff
@@ -182,6 +182,17 @@ bool siwx91x_param_changed(struct wifi_iface_status *prev_params,
 	return false;
 }
 
+static int siwx91x_set_max_tx_power(void)
+{
+	sl_wifi_interface_t interface = sl_wifi_get_default_interface();
+	sl_wifi_max_tx_power_t max_tx_power = {
+		.scan_tx_power = DT_PROP(DT_NODELABEL(wifi0), wifi_max_tx_pwr_scan),
+		.join_tx_power = DT_PROP(DT_NODELABEL(wifi0), wifi_max_tx_pwr_join),
```

Comment:
Let's try to avoid to rely on the node labels. I believe this should work: `DT_INST(0, wifi_max_tx_pwr_scan)`.

We also try to group all the DT reference in the definition of `sidev` structure, but this is not a big deal (BTW, this struct should be split in two parts: `config` and `data`. Config could live in the flash, so we could save some memory).